risc0/zkp/accel/backend/cpu/impl.cpp

```diff
@@ -130,7 +130,7 @@ void batchEvaluateAny(AccelConstSlice<Fp> coeffs,
     size_t id = reinterpret_cast<const uint32_t*>(which.buf()->buf)[i];
     Fp4 x = reinterpret_cast<const Fp4*>(xs.buf()->buf)[i];
     const Fp* coeffLocal = coeffsPtr + (1 << po2) * id;
-    for (size_t i = 0; i < (1 << po2); i++) {
+    for (size_t i = 0; i < (1U << po2); i++) {
```

Comment:
Should this be 1U, or should it be (size_t)1?

Reply:
In c++23 we can use "1z" to get a literal of type size_t, but I guess for now we can cast it manually.